examples/typescript-graphql-request/src/main.ts

```diff
@@ -46,6 +46,3 @@ export const getPeople = async (first?: number) => {
   }
   return res?.allPeople?.edges;
 };
-
-getPeople().then(res => console.log(res));
-getPeople(10).then(res => console.log(res));
```

Comment:
These calls cause the Jest tests to fail. These also look like debugging statements that were never removed 🤔